packages/ra-core/src/controller/field/ReferenceManyFieldController.js

```diff
@@ -135,7 +135,7 @@ export class ReferenceManyFieldController extends Component {
             currentSort: this.state.sort,
             data,
             ids,
-            isLoading: typeof ids === 'undefined',
+            loadedOnce: typeof ids !== 'undefined',
```

Comment:
this can be considered a BC break, but it was never documented, so I'm confident it should not break much.